internal/web/routes/routes.go

```diff
@@ -0,0 +1,18 @@
+package routes
+
+import (
+	"context"
+
```

Comment:
Validate route parameters within the handler functions to prevent potential security vulnerabilities such as injection attacks or unintended behavior due to unexpected input.